Dockerfile

```diff
@@ -1,9 +1,20 @@
-FROM golang:alpine as builder
-RUN apk --update --no-cache add bash
+FROM golang:1.14 as builder
 WORKDIR /app
 ADD . .
 RUN go build -o app
 
+##########################
+
+FROM builder as dev
+
+COPY bashrc /root/.bashrc
+
+RUN go get github.com/codegangsta/gin && \
```

Comment:
`okteto/go` has there and the bash, maybe we can use that as the base?

Reply:
I prefer this way because:
- Your dev image has the go cache and the source in the image itself.
- It shows a good practice on how to define and manage your own dev images.